troposphere/templates/index.html

```diff
@@ -13,10 +13,9 @@
     <!--
     <link rel="stylesheet" href="{{ BASE_URL }}/{{ THEME_URL }}/theme.css" rel="stylesheet" type="text/css">
     -->
-
-    <!-- Adobe Typekit (for Myriad Pro) -->
-    <script type="text/javascript" src="//use.typekit.net/gty2jpo.js"></script>
-    <script type="text/javascript">try{Typekit.load();}catch(e){}</script>
+    
+    <!-- Fonts -->
+    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,700,300' rel='stylesheet' type='text/css'>
```

Comment:
Okay, I was wrong. The template that `airport_ui` is rendered _into_ is `cf2.html` - **not** `index.html`. 

We are good :white_check_mark: